frontend/src/containers/Routing/index.tsx

```diff
@@ -7,11 +7,12 @@ import { IAppState } from '../../common/models/store/IAppState';
 import { fetchUserRoutine } from '../../scenes/Auth/routines';
 import PrivateRoute from '../PrivateRoute';
 import PublicRoute from '../PublicRoute';
-import Resources from '../../scenes/Resources/index';
+// import Resources from '../../scenes/Resources/index';
 import Apps from '../../scenes/Apps/index';
 import Auth from '../../scenes/Auth/containers';
 import { IBindingAction } from '../../common/models/callback/IBindingAction';
 import Settings from '../../scenes/Settings';
+import ResourceRouting from '../../scenes/Resources/containers/Routing/index';
```

Comment:
if we use Routing component here instead of the scene's root file, the Routing component should become the root